demos/basics/loading-pandas.ipynb

```diff
@@ -214,7 +214,7 @@
        "4      a      c"
```

Comment:
<p>This doesn't put together the edge weights from prev section?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1397/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>

Reply:
Good idea.